test/atomic_enum_test.rb

```diff
@@ -0,0 +1,35 @@
+require 'test_helper'
+
+class TestAtomicEnum < MiniTest::Unit::TestCase
```

Comment:
Actually, if you look at the naming of all the test files in this folder, it's written like this.